src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -34,8 +35,19 @@ const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
   const deviceHeaders = [
     { content: "Name", sortKey: "name", className: "u-text--muted" },
     { content: "Type", sortKey: "type", className: "u-text--muted" },
+    { content: "Details", sortKey: "type", className: "u-text--muted" },
```

Comment:
`sortKey` here should be `details`

Reply:
We can also avoid the sort key, so the column is not sortable. It might be hard and not relevant to sort by the details.